src/ray/raylet/test/local_object_manager_test.cc

```diff
@@ -634,7 +646,7 @@ TEST_F(LocalObjectManagerTest, TestSpillUptoMaxFuseCount) {
     objects.push_back(std::move(object));
   }
   manager.PinObjectsAndWaitForFree(object_ids, std::move(objects), owner_address);
-  ASSERT_TRUE(manager.SpillObjectsOfSize(total_size));
+  ASSERT_TRUE(manager.SpillObjectsOfSize(total_size, MAX_SPILLING_SIZE));
```

Comment:
@rkooo567 Two questions: 1. why is this test in `LocalObjectManagerTest` but not `LocalObjectManagerFusedTest`? I thought the former sets max_fused_count to 1 whereas the latter sets `max_fused_count` to 15, which is what is tested here. 2. Why does line 657 say "we limit the max fused number of objects to be 10"? Is this intended to be 15?